### PR TITLE
tiup: refine description about `tiup playground --tag`

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/19669
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -99,8 +104,6 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
         > **注意：**
         >
         > + 支持 v5.2.0 及以上版本的 TiDB 在 Apple M1 芯片的机器上运行 `tiup playground`。
```
Comment (hfxsd) [inline quick-start-with-tidb.md:106]:
```suggestion
        > 支持 v5.2.0 及以上版本的 TiDB 在 Apple M1 芯片的机器上运行 `tiup playground`。
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -68,6 +68,11 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
 
 3. 在当前 session 执行以下命令启动集群。
 
+    > **注意：**
+    >
+    > + 以这种方式执行的 playground，在结束部署测试后 TiUP 会清理掉原集群数据，重新执行该命令后会得到一个全新的集群。
+    > + 若希望持久化数据，可以执行 TiUP 的 `--tag` 参数：`tiup --tag <your-tag> playground ...`，详情参考[启动集群时指定 `tag` 以保留数据](/tiup/tiup-playground.md#启动集群时指定-tag-以保留数据)。
```
Comment (hfxsd) [inline quick-start-with-tidb.md:74]:
```suggestion
    > + 若希望持久化数据，可以执行 TiUP 的 `--tag` 参数：`tiup playground --tag ${tag_name}`，详情参考[启动集群时指定 `tag` 以保留数据](/tiup/tiup-playground.md#启动集群时指定-tag-以保留数据)。
```

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -185,6 +188,11 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
...
```
Comment (hfxsd) [inline quick-start-with-tidb.md:194]:
```suggestion
    > + 若希望持久化数据，可以执行 TiUP 的 `--tag` 参数：`tiup playground --tag ${tag_name}`，详情参考[启动集群时指定 `tag` 以保留数据](/tiup/tiup-playground.md#启动集群时指定-tag-以保留数据)。
```

**Review by xhebox — APPROVED** [review]

File: quick-start-with-tidb.md [hunk 4]
```diff
@@ -185,6 +188,11 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
 
 3. 在当前 session 执行以下命令启动集群。
 
+    > **注意：**
+    >
+    > + 以这种方式执行的 playground，在结束部署测试后 TiUP 会清理掉原集群数据，重新执行该命令后会得到一个全新的集群。
+    > + 若希望持久化数据，可以执行 TiUP 的 `--tag` 参数：`tiup playground --tag ${tag_name}`，详情参考[启动集群时指定 `tag` 以保留数据](/tiup/tiup-playground.md#启动集群时指定-tag-以保留数据)。
```
Comment (lilin90) [inline quick-start-with-tidb.md:194]:
```suggestion
    > 按以下方式执行的 playground，在结束部署测试后 TiUP 会清理掉原集群数据，重新执行该命令后会得到一个全新的集群。若希望持久化数据，需要添加 TiUP 的 `--tag` 参数：`tiup playground --tag ${tag_name}`，详见[启动集群时指定 `tag` 以保留数据](/tiup/tiup-playground.md#启动集群时指定-tag-以保留数据)。
```

File: tiup/tiup-playground.md [hunk 5]
```diff
@@ -24,52 +24,10 @@ tiup playground ${version} [flags]
...
-      --tiproxy.host host          Playground 的 TiProxy host。如果没有提供，TiProxy 会使用 host 参数作为它的 host
-      --tiproxy.port int           Playground 的 TiProxy 端口。如果没有提供，TiProxy 会使用 6000 作为它的端口
-      --tiproxy.timeout int        TiProxy 最长等待超时时间，单位为秒，若配置为 0，则永不超时（默认为 60）。
-      --tso int                    设置集群中 TSO 节点的数量（默认为 1），只能在 pd.mode 为 'ms' 的时候设置
-      --tso.host host              指定 TSO 节点的监听地址
-      --tso.binpath string         指定 TSO 节点上二进制文件的位置（开发调试用，可忽略）
-      --tso.config string          指定 TSO 节点的配置文件（开发调试用，可忽略）
-  -v, --version                    显示 playground 的版本号
-      --without-monitor            设置不使用 Prometheus 和 Grafana 的监控功能。若不添加此参数，则默认开启监控功能。
+可以使用下面命令查看 playground 的命令行参数说明：
```
Comment (lilin90) [inline tiup/tiup-playground.md:27]:
```suggestion
可以使用以下命令查看 playground 的命令行参数说明：
```

File: quick-start-with-tidb.md [hunk 6]
```diff
@@ -68,6 +68,10 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
 
 3. 在当前 session 执行以下命令启动集群。
 
+    > **注意：**
+    >
+    > 按以下方式执行的 playground，在结束部署测试后 TiUP 会清理掉原集群数据，重新执行该命令后会得到一个全新的集群。若希望持久化数据，需要添加 TiUP 的 `--tag` 参数：`tiup playground --tag ${tag_name}`，详见[启动集群时指定 `tag` 以保留数据](/tiup/tiup-playground.md#启动集群时指定-tag-以保留数据)。
```
Comment (lilin90) [inline quick-start-with-tidb.md:73]:
```suggestion
    > 如果按以下方式执行 playground，在结束部署测试后，TiUP 会自动清理掉原集群数据，重新执行命令会得到一个全新的集群。若希望持久化数据，需要在启动集群时添加 TiUP 的 `--tag` 参数：`tiup playground --tag ${tag_name}`，详见[启动集群时指定 `tag` 以保留数据](/tiup/tiup-playground.md#启动集群时指定-tag-以保留数据)。
```

File: quick-start-with-tidb.md [hunk 7]
```diff
@@ -185,6 +187,10 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
...
```
Comment (lilin90) [inline quick-start-with-tidb.md:192]:
```suggestion
    > 如果按以下方式执行 playground，在结束部署测试后，TiUP 会自动清理掉原集群数据，重新执行命令会得到一个全新的集群。若希望持久化数据，需要在启动集群时添加 TiUP 的 `--tag` 参数：`tiup playground --tag ${tag_name}`，详见[启动集群时指定 `tag` 以保留数据](/tiup/tiup-playground.md#启动集群时指定-tag-以保留数据)。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**